web/js/front.js

```diff
@@ -1,25 +1,39 @@
-document.getElementById('pc').focus();
-
 (function(){
-    var around_forms = document.querySelectorAll('form[action*="around"]');
-    for (var i=0; i<around_forms.length; i++) {
-        var form = around_forms[i];
+    var each = function(selector, callback) {
+        var elements = document.querySelectorAll(selector);
+        for (var i=0; i<elements.length; i++) {
+            callback(elements[i]);
+        }
+    };
+
+    var goToSearch = function(e){
+        if (e) { e.preventDefault(); }
+        var form = document.querySelectorAll('#postcodeForm')[0];
+        if (form) {
+            window.scroll(0, form.getBoundingClientRect().top + window.scrollY);
```

Comment:
I don't understand what the scrollY is doing here. Would `0` be good enough, even on small screens?